src/parkapi_sources/converters/vrn_p_r/models.py

```diff
@@ -0,0 +1,174 @@
+    capacity_family: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    capacity_woman: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    capacity_bus: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    capacity_truck: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    capacity_carsharing: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    capacity_disabled: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    max_height: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    has_realtime_data: OptionalUnset[bool] = (
+        NoneToUnsetValue(MappedBooleanValidator(mapping={'ja': True, 'nein': False})),
+        DefaultUnset,
+    )
+    vrn_sensor_id: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    realtime_opening_status: OptionalUnset[VrnParkAndRidePropertiesOpeningStatus] = (
+        NoneToUnsetValue(EnumValidator(VrnParkAndRidePropertiesOpeningStatus)),
+        DefaultUnset,
+    )
+    has_lighting: OptionalUnset[bool] = (
+        NoneToUnsetValue(MappedBooleanValidator(mapping={'ja': True, 'nein': False})),
+        DefaultUnset,
+    )
+    has_fee: OptionalUnset[bool] = (
+        NoneToUnsetValue(MappedBooleanValidator(mapping={'ja': True, 'nein': False})),
+        DefaultUnset,
+    )
+    is_covered: OptionalUnset[bool] = (
+        NoneToUnsetValue(MappedBooleanValidator(mapping={'ja': True, 'nein': False})),
+        DefaultUnset,
+    )
+    related_location: OptionalUnset[str] = NoneToUnsetValue(StringValidator(min_length=0, max_length=256)), DefaultUnset
+    opening_hours: OptionalUnset[str] = NoneToUnsetValue(StringValidator(min_length=0, max_length=256)), DefaultUnset
+    park_and_ride_type: OptionalUnset[VrnParkAndRidePRType] = (
+        NoneToUnsetValue(EnumValidator(VrnParkAndRidePRType)),
+        DefaultUnset,
+    )
+    max_stay: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    fee_description: OptionalUnset[str] = NoneToUnsetValue(StringValidator(max_length=512)), DefaultUnset
+    realtime_free_capacity: int = IntegerValidator(min_value=0)
+    realtime_occupied: int = IntegerValidator(min_value=0)
+    reatime_data_updated: OptionalUnset[datetime] = (
+        NoneToUnsetValue(TimestampDateTimeValidator(allow_strings=True, divisor=1000)),
+        DefaultUnset,
+    )
+
+
+@validataclass
+class VrnParkAndRideFeaturesInput:
+    geometry: GeojsonFeatureGeometryInput = DataclassValidator(GeojsonFeatureGeometryInput)
+    properties: VrnParkAndRidePropertiesInput = DataclassValidator(VrnParkAndRidePropertiesInput)
+
+    def to_static_parking_site_input(self) -> StaticParkingSiteInput:
+        return StaticParkingSiteInput(
+            uid=str(self.properties.vrn_sensor_id),
```

Comment:
I don't see any groupable information in the original dataset. What do you try to group?